sqldb/sqlite.go

```diff
@@ -58,7 +56,7 @@ type SqliteStore struct {
 
 // NewSqliteStore attempts to open a new sqlite database based on the passed
 // config.
-func NewSqliteStore(cfg *SqliteConfig) (*SqliteStore, error) {
+func NewSqliteStore(cfg *SqliteConfig, dbPath string) (*SqliteStore, error) {
```

Comment:
Why not path in config?

Reply:
So it'll currently just create the `lnd.sqlite` file in the chandb path to reflect the semantics of the existing configs (see also in `lncfg/db.go`).